.gitlab-ci.yml

```diff
@@ -163,6 +163,12 @@ test/ubuntu-python3:cuda-9.0:
 test/ubuntu-python3:cuda-10.1:
   extends: .test_template
   needs: ["ubuntu-python3:cuda-10.1:build"]
+  script:
```

Comment:
Why did you introduce these changes? Is that related to requests?

Reply:
here I'm testing that sphinx can actually run; we merged the previous PR without realizing the sphinx executables installed in the user directory were not accessible in the PATH, causing the CI failures in the espresso repo when building the sphinx docs